lib/features/email/presentation/email_controller.dart

```diff
@@ -221,8 +259,13 @@ class EmailController extends BaseController with AppLoaderMixin {
   void _getEmailContentAction(EmailId emailId) async {
     final accountId = mailboxDashBoardController.accountId.value;
     final baseDownloadUrl = mailboxDashBoardController.sessionCurrent?.getDownloadUrl();
+    final listEmailIdLoaded = presentationEmailsLoaded.map((e) => e.emailCurrent!.id).toList();
+    EmailLoaded? emailLoaded;
+    if(listEmailIdLoaded.contains(_currentEmailId)) {
+      emailLoaded = presentationEmailsLoaded.firstWhere((e) => e.emailCurrent!.id == _currentEmailId);
```

Comment:
should `try/catch` at here. because it will throw exception when not found email